### PR TITLE
Use nncf_graph for FBC

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
### Changes

- Use compressed graph for torch backend
- Add `with_bias` for `WeightedLayerAttributes` what means that layer contain bias inside module (will true only for torch and tf)
- Removed `bias` from `LinearLayerAttributes`, use `with_bias`.
- Remove `model` from functions backend of `fast_bias_correction`
 
### Related tickets
111576

### CODE REVIEW

**Comment by AlexanderDokuchaev** [comment]
job post_training_quantization: 113

File: nncf/torch/model_transformer.py [hunk 1]
```diff
@@ -139,7 +139,8 @@ def update_fused_bias(target_node_name: str, new_bias: Tensor, model: NNCFNetwor
     :param new_bias: New bias value.
     :param model: The model.
     """
-    fused_node = get_potential_fused_node(target_node_name, model)
+    nncf_graph = model.get_graph()
```
Comment (KodiaqQ) [inline nncf/torch/model_transformer.py:142]: Wouldn't it be very expensive to `get_graph` for every `update_fused_bias` call?

File: nncf/torch/quantization/quantize_functions.py [hunk 2]
```diff
@@ -20,6 +20,8 @@
 from nncf.torch.quantization.extensions import QuantizedFunctionsCUDA
 from nncf.torch.utils import add_domain
 
+QUANTIZE_OPERATION_NAMES = ["symmetric_quantize", "asymmetric_quantize"]
```
Comment (KodiaqQ) [inline nncf/torch/quantization/quantize_functions.py:23]: Why is it here? Where is it used?

File: nncf/torch/model_transformer.py (hunk 1 above)
Comment (AlexanderDokuchaev) [reply nncf/torch/model_transformer.py:142]: No, it's nncfnetwrok contain `_compressed_graph` to save graph, and does not build graph on each call. https://github.com/openvinotoolkit/nncf/blob/develop/nncf/torch/nncf_network.py#L425

**Review by KodiaqQ — APPROVED** [review]

**Comment by AlexanderDokuchaev** [comment]
run pylint pre-commit tests

**Review by alexsu52 — APPROVED** [review]
Cool staff! Thanks!

**Pull request merged**